numpy/core/tests/test_multiarray.py

```diff
@@ -4540,6 +4542,23 @@ def test_roundtrip_filename(self):
         y = np.fromfile(self.filename, dtype=self.dtype)
         assert_array_equal(y, self.x.flat)
 
+    def test_roundtrip_pathlib(self):
+        if sys.version_info[:2] >= (3, 4):
```

Comment:
Better handled as:
```
try:
    import pathlib
import ImportError:
    pathlib = None
```
then
```
@pytest.mark.skipif(pathlib is None)
def test_roundtrip_pathlib(self):
    # the test
```

Reply:
Done.
Your feedback is much appreciated. Thank you.